interface/orders/receive_hl7_results.inc.php

```diff
@@ -761,6 +754,10 @@ function receive_hl7_results(&$hl7, &$matchreq, $lab_id = 0, $direction = 'B', $
             $tmp = explode($d2, $a[4]);
             $in_procedure_code = $tmp[0];
             $in_procedure_name = $tmp[1];
+            //add specimen source onto name for dermatology
+            if (!empty($a[14])) {
+                $in_procedure_name = $in_procedure_name . " - " . $a[15];
```

Comment:
Do you really mean to append $a[15] if $a[14] is not empty?

Reply:
nope, thanks